content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -408,6 +408,7 @@ The following table below lists configuration variables that are available for b
 | `DD_TRACE_GLOBAL_TAGS`<br/><br/>`GlobalTags`        | If specified, adds all of the specified tags to all generated spans.                                                                                                                                              |
 | `DD_TRACE_DEBUG`                                    | Enables or disables debug logging. Valid values are: `true` or `false` (default).                                                                                                                                 |
 | `DD_TRACE_HEADER_TAGS`                              | Accepts a map of case-insensitive header keys to tag names and automatically applies matching header values as tags on root spans. (e.g. : `CASE-insensitive-Header:my-tag-name,User-ID:userId`). Available for version 1.18.3+  |
+| `DD_TRACE_SERVICE_MAPPING`                          | Rename services using configuration. Accepts a map of service name keys to rename, and the name to use instead, in the format `[from-key]:[to-name]` e.g. : `mysql:main-mysql-db, mongodb:offsite-mongodb-service`. `from-key` is specific to the integration type, and should exclude the application name prefix. For example, to rename `my-application-sql-server` to `main-db`, use `sql-server:main-db`. Added in version 1.23.0 |
```

Comment:
Will this setting also be available to customers via the NuGet API? If so can we please also add that setting similar to how we do for DD_TRACE_GLOBAL_TAGS? 

Overall the text seems good - a bit wordy but can we trim it down at all?

Reply:
It's not available in _quite_ the same way as `GlobalTags`,  but it is settable via `SetServiceNameMappings()` so I'll add that. 

The issue with trimming it is clarity - I'm happy to reduce it (it was briefer in [the first iteration](https://github.com/DataDog/documentation/pull/9502/commits/41572048e1ad65af224f1ac998c37f2ed5360678)). I added extra clarification about what the `from-key` needs to be, as this is different to the Java implementation. Also, could remove the examples to make it briefer (but I personally like examples in docs 🙂)

Reply:
Could make this slightly shorter by removing the first set of examples:

> Rename services using configuration. Accepts a map of services to rename, in the format `[from-key]:[to-name]`. The `from-key` is specific to the integration type, and should exclude the application name prefix. For example, to rename `my-application-sql-server` to `main-db`, use `sql-server:main-db`. Added in version 1.23.0`

Reply:
The original looks good to me - we can adjust later if needed.